src/js/components/AppConfigEditFormComponent.jsx

```diff
@@ -160,6 +160,12 @@ var AppConfigEditFormComponent = React.createClass({
     });
   },
 
+  hasVIP: function () {
+    var features = this.props.features;
+
+    return Util.isArray(features) && features.includes("vips");
```

Comment:
I've tested it.
Passing `null` to the features-prop _doesn't_ trigger the default props!
And I receive a `features is null` error by the `includes`-function.
So the _isArray-check is necessary_, because features could be `null` eventually.